### ISSUE TITLE
PHP feature not working with version 7.4.33

### ISSUE THREAD

**Issue by gremo** [devcontainers/features#426]
A bounch of warnings, can't find the exact error, any help is much appreciated

```
#0 68.43       |                                         ^~~~~~~~~~~~~~~~~~~~~~~
#0 68.43 In file included from /usr/src/php/ext/openssl/openssl.c:46:
#0 68.43 /usr/include/openssl/rsa.h:289:29: note: expected 'RSA *' {aka 'struct
rsa_st *'} but argument is of type 'const struct rsa_st *'
#0 68.43   289 |                        RSA *rsa, int padding);
#0 68.43       |                        ~~~~~^~~
#0 68.48 make: *** [Makefile:640: ext/openssl/openssl.lo] Error 1
#0 68.48 make: *** Waiting for unfinished jobs....
#0 88.17 ERROR: Feature "PHP" (ghcr.io/devcontainers/features/php) failed to ins
tall! Look at the documentation at https://github.com/devcontainers/features/tre
e/main/src/php for help troubleshooting this error.
------
ERROR: failed to solve: executor failed running [/bin/sh -c cd /tmp/build-featur
es/php_1 && chmod +x ./devcontainer-features-install.sh && ./devcontainer-featur
es-install.sh]: exit code: 2
```

**Comment by jkeech** [comment]
@gremo can you please share the devcontainer you are using or provide a minimal repro?

**Comment by gremo** [comment]
Sure, it' simply as:

```json
{
    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
    "features": {
        "ghcr.io/devcontainers/features/php:1": {
            "version": "7.4.33"
        }
    }
}

```

**Comment by samruddhikhandale** [comment]
@gremo Php v7.4 has reached end of life in Dec 2021. Do you mind switching to the latest supported versions and give that a try?

https://www.php.net/supported-versions.php

**Comment by gremo** [comment]
I know @samruddhikhandale but being able to use older PHP versions (legacy projects) it's one of the advances of containers, isn't it? 

It works well with latest version. It would be great to also make it work for older PHP versions too.

**Comment by IpstanKabul** [comment]
> @gremo Php v7.4 has reached end of life in Dec 2021. Do you mind switching to the latest supported versions and give that a try?
> 
> https://www.php.net/supported-versions.php

I'm not really sure I understand the logic here. Docker doesn't limit the image versions to current, why is this feature limiting us? As a workaround, I'm installing php via a postInstallScript but it would be good if the feature had flexibility built-in.

**Comment by gremo** [comment]
@IpstanKabul thanks for sharing the same feelings. As a workaround I ended up using official php images with some custom features. Here https://github.com/gremo/devcontainers-features if you want to take a look.

As I explained, I need maintain (at work) a lot of legacy projects even written in php 5.6. They cannot be updated due to money and time constraints. 

Closing this as it wont’t be resolved I think.
